### ISSUE TITLE
Integration with github-icons.com

### ISSUE THREAD

**Issue by samdenty** [Idered/iridium#28]
Hi there 👋 

I'm the creator of https://github-icons.com / https://github.com/samdenty/github-icons.

We provide a free-to-use API to retrieve icons for NPM packages. I think it'd be pretty cool if you integrated it with this project

<img width="351" alt="CleanShot 2022-12-24 at 19 54 41@2x" src="https://user-images.githubusercontent.com/13242392/209449525-557360bc-be1f-4fa5-9d0c-52c5a2862d09.png">

The API is as simple as loading an image tag with

```
https://github-icons.com/npm/PACKAGE_NAME
```

for example `https://github-icons.com/npm/react`
![react](https://github-icons.com/npm/react)

This image can return a 404 if it's not found. And also it can take a while waiting for the GitHub API (second time is always instant), so on https://github-icons.com I do the following.

1. Render the `<img>` tag with `opacity: 0`, and `object-fit: contain` (so it retains aspect ratio)
2. Render a loading animation in the place of the icon, with position absolute
3. Attach an onLoad handler to the image, which removes the loading animation and unhides the icon
4. Attach an onError handler to the image which updates the `src` to either a generic npm logo or not found icon

**Comment by Idered** [comment]
Hey @samdenty, sorry for late response 😅

Here are a few problems I see:
- showing author face 
- size consistency 
- blurry/pixelated icons
- eslint logo has white background

Overall I like your project but I think that results are not refined enough. 

I consider creating more robust solution powered by community suggestions

<img width="561" alt="CleanShot 2023-07-01 at 11 30 55@2x" src="https://github.com/Idered/iridium/assets/230404/99add147-3b39-4d3b-b3f6-5dcec5194117">

---

**Comment by samdenty** [comment]
Yeah some icons could do with custom picked ones instead. I was testing the water as I hadn't seen any project that scraped github repo icons automatically before and thought it would be a good idea.

It's got some users now around 700 daily, but i'd like to see it grow.

Maybe I should add a curating / icon suggestion thing, instead of relying on package authors to manually set the `"icon": "./icon.png"` field in package json to use custom icons. I was picking between both ideas and thought it could be nicer to have a standard, but i guess some repos will never accept new contributions.

Regarding showing the NPM avatar on icons, I did that as that's what github does on repos. I prioritise organization avatars over user ones, but maybe I should not display user ones at all when on an NPM package. Curious on your thoughts
